modules/manage/pages/manage-nodes/create-cluster.adoc

```diff
@@ -209,7 +209,7 @@ Note that the default disk-paths for data, indexes, and analytics will be used,
 xref:manage:manage-nodes/initialize-node.adoc#initialize-node-with-the-cllii[Initialize a Node with the CLI].)
```

Comment:
It looks like the section id has a typo "initialize-node-with-the-cllii". I am not sure if there are many xrefs to this section, hence request you to check and fix all instances. Thanks.

Reply:
Thanks, have fixed.